testcases/ock_tests.sh.in

```diff
@@ -249,15 +249,17 @@ run_tests()
         OCK_TESTS="$OCK_TEST"
     fi
     echo "***** Will run the following tests for slot $1: $(ls -U $OCK_TESTS)"
+    ALLRES=0
     for j in $( ls -U $OCK_TESTS ); do
         echo "** Now executing '$j'"
         $j -slot $1 $NO_STOP 2>&1
         RES=$?
-		if [ $RES -ne 0 ]; then
-			echo "ERROR: Testcase $j failed to execute."
-			exit $RES
-		fi
+        if [ $RES -ne 0 ]; then
+            ALLRES=$RES
+            echo "ERROR: Testcase $j failed to execute, rc: $RES"
```

Comment:
Couldn't that produce garbage when multiple concurrent processes echo something into the same file?

Reply:
Possibly yes.  How about creating a file `error_file.$j.$slot` and check with `ls error_file.*.* &>/dev/null` if there were errors.  Then we could just `cat error_file.*.*` at the end.

Reply:
Good idea!  Just force pushed the last commit with your suggestion implemented. Please take a look.

Reply:
That is nice.  I think this is the best solution.  I resolved my other comments around this.

Reply:
Did you check if this would somehow break our CI, e.g. due to the additional output ?

Reply:
The CI grabs the `log-slot_*.log` files and parses them.  The other files and output are ignored.  I did not see any change that puts something new the these files (all lines from the current regress.h are already known to the CI).

Reply:
Thinking about this change, we could even simplify and reduce the number of error files.  Only slots run in parallel.  We could just create one error file per slot instead of one per failing test and slot.  But the current implementation should work as well.

Reply:
Right. Changed it to use only one error file per slot.